src/pages/join-us.astro

```diff
@@ -1,138 +1,138 @@
 ---
-import Layout from "~/layouts/PageLayout.astro";
-import Header from "~/components/widgets/Header.astro";
+import Layout from '~/layouts/PageLayout.astro';
+import Header from '~/components/widgets/Header.astro';
 import Prices from '~/components/widgets/Pricing.astro';
 import FAQs from '~/components/widgets/FAQs.astro';
 
-import { headerData } from "~/data";
+import { headerData } from '~/data';
 
 const meta = {
-  title: "PLEBNET.DEV Join Us",
+  title: 'PLEBNET.DEV Join Us',
 };
-
 ---
+
 <Layout {meta}>
   <Fragment slot="header">
     <Header
       {...headerData}
       actions={[
         {
-          type: "orange",
-          text: "Shop",
-          href: "https://plebnet-dev.printify.me/",
+          type: 'orange',
+          text: 'Shop',
+          href: 'https://plebnet-dev.printify.me/',
         },
       ]}
       isSticky
     />
   </Fragment>
 
-
-<Prices
-  title="Join Us"
-  subtitle="Individual or Organizations welcome! Questions? See FAQ below."
-  prices={[
-    {
-      title: 'Individual',
-      subtitle: 'Optimal choice for personal use',
-      price: 100000,
-      period: 'sats per year',
-      items: [
-        {
-          description: 'Join Members only Events',
-        },
-        {
-          description: 'Access to members only Discord',
-        },
-        {
-          description: 'Access to Member only resources',
-        },
-        {
-          description: 'Mentor or get access to mentoring',
-        },
-      ],
-      callToAction: {
-        targetBlank: false,
-        text: 'Get started',
-        href: '/Indiv',
+  <Prices
+    title="Join Us"
+    subtitle={[
+      'Individual or Organizations welcome! Questions? See FAQ below.',
+      '<br /><a href="https://discord.gg/mX94kNTcRp" class="discord-link">Join our Discord</a>',
```

Comment:
Don't hardcode the link. We just did permalinks, please use that!! Remember?

Also its not obvious the Discord is a link here, as its just text. Use a style that matches the rest of the template to indicate that "Discord" is a link.  move "Join Our" to regular text. 

it might be require a little more tweaking here, as its inside of the <Prices> component and not just straight HTML. Have a look and if you get stuck let me know

Reply:
Updated discord permalink!

Reply:
This link is still hardcoded in your updated PR